{"EventId":0,"LogLevel":"Information","Category":"Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageSub","Message":"PreHook:TestMessageSub,test-message-id,test-trace-id","State":{"Message":"PreHook:TestMessageSub,test-message-id,test-trace-id","MessageType":"TestMessageSub","MessageId":"test-message-id","TraceId":"test-trace-id","{OriginalFormat}":"PreHook:{MessageType},{MessageId},{TraceId}"}}
{"EventId":0,"LogLevel":"Error","Category":"Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageSub","Message":"ExceptionHook:TestException,test-message-id,test-trace-id","Exception":"Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages\u002BTestException: test exception\n   at Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageBaseHandler.Handle(TestMessageBase message, CancellationToken cancellationToken) in {ProjectDirectory}Messaging/LoggingMiddlewareTestMessages.cs:line 920\n   at Conqueror.Middleware.Logging.Messaging.LoggingMessageMiddleware\u00602.Execute(MessageMiddlewareContext\u00602 ctx) in {SolutionDirectory}src/Conqueror.Middleware.Logging/Messaging/LoggingMessageMiddleware.cs:line 52\n   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine\u0026 stateMachine)\n   at Conqueror.Middleware.Logging.Messaging.LoggingMessageMiddleware\u00602.Execute(MessageMiddlewareContext\u00602 ctx)\n   at Conqueror.Messaging.MessagePipelineRunner\u00602.\u003C\u003Ec__DisplayClass3_2.\u003CExecute\u003Eb__1(TMessage message, CancellationToken token) in {SolutionDirectory}src/Conqueror/Messaging/MessagePipelineRunner.cs:line 36","State":{"Message":"ExceptionHook:TestException,test-message-id,test-trace-id","ExceptionType":"TestException","MessageId":"test-message-id","TraceId":"test-trace-id","{OriginalFormat}":"ExceptionHook:{ExceptionType},{MessageId},{TraceId}"}}
